conf/machine/freedom-u540.conf

```diff
@@ -10,7 +10,7 @@ MACHINE_FEATURES = "screen keyboard ext2 ext3 serial"
 KERNEL_IMAGETYPE = "vmlinux"
 
 PREFERRED_PROVIDER_virtual/kernel ?= "linux-riscv"
-PREFERRED_VERSION_linux-riscv ?= "4.15%"
+PREFERRED_VERSION_linux-riscv ?= "4.17%"
```

Comment:
do we have all the drivers (serial etc.) in 4.17 now ? I think this change would need a bit of testing